SlotMachine.hs

```diff
@@ -35,7 +36,7 @@ slotMachine1 = SlotMachine {
         ("%%%%", 40), ("%%%%%", 150),
         ("$$", 2), ("$$$", 20), ("$$$$", 200), ("$$$$$", 1000)
     ],
-    spreads = [ ('#', 3, 50.0), ('#', 4, 100) ], -- [(Char, Int, Double)]
+    spreads = [ ('#', 3, 50), ('#', 4, 100) ], -- [(Char, Int, Double)]
```

Comment:
An integer literal has the polymorphic type Num a => a which will automatically resolve here to Double thanks to type inference.